real_estate/real_estate/doctype/property_booking_order/property_booking_order.py

```diff
@@ -6,11 +6,16 @@
 from frappe.model.document import Document
 from frappe.utils import cint, getdate, flt, add_to_date, today
 from real_estate.real_estate.doctype.property_payment_plan_template.property_payment_plan_template import get_payment_plan
+from erpnext.accounts.party import get_address_display
+from frappe.contacts.doctype.address.address import get_default_address
+from frappe.contacts.doctype.contact.contact import get_default_contact
+from frappe.contacts.doctype.contact.contact import get_contact_details
 import json
 
 
 force_fields = [
 	'unit_number', 'property_type', 'project', 'block', 'block_name', 'floor',
+	'tax_cnic', 'customer_address_name', 'customer_address', 'contact_person', 'phone_no', 'mobile_no', 'contact_email'
```

Comment:
do not force `customer_address_name` and `contact_person` ... user should be able to change customer's address and contact person instead of being forced to use the default values only